.gitignore

```diff
@@ -2,3 +2,6 @@
 *~
 *.car
 *.out
+.terraform
+.vscode
+*.code-workspace
```

Comment:
Look at the above changes `.vscode` and `.code-worskace` look IDE specifc?
if so, please avoid IDE specific gitignore entries. Instead, consider setting up a global gitignore on your machine that filters out all such files across every repo for the IDE you use.